commands/push.py

```
@@ -0,0 +1,283 @@
+    run_tests,
+)
+
+
+def get_git_status():
+    """Get current git status"""
+    try:
+        result = subprocess.run(
+            ["git", "status", "--porcelain"], capture_output=True, text=True, check=True
+        )
+        return result.stdout.strip()
+    except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)